tests/integration/cli/test_cli.py

```diff
@@ -273,6 +276,8 @@ def test_otx_explain_e2e(
     """
     if "tile" in recipe:
         pytest.skip("Explain is not supported for tiling yet.")
+    if "anomaly" in recipe:
+        pytest.skip("Explain is not supported for anomaly yet.")
```

Comment:
What is explain exactly? Is it the saliency map? The current Anomaly tasks in OTX always produce anomaly maps even if they don't return them for classification task. Maybe explain might not apply to it.

Reply:
It would be great to discuss this with Vlad in the next PR.